source/lumars/state.d

```diff
@@ -688,6 +688,15 @@ struct LuaState
         import std.conv : to;
         import std.traits : isNumeric, isDynamicArray, isAssociativeArray, isDelegate, isPointer, isFunction,
                             PointerTarget, KeyType, ValueType, FieldNameTuple, TemplateOf, TemplateArgsOf;
+        
+        static if (__traits(compiles, value = null))
```

Comment:
```suggestion
        static if (__traits(compiles, value is null))
```
(If that works).

My concern is that custom types can support `value = null` without `value is null` being valid for them as I believe you can't override the `is` operator.